### DISCUSSION TITLE
Help Pls

### DISCUSSION THREAD

**Discussion by deborahsv** [community/community#120878] [Programming Help] [answered]
Please, i can't see my code with the css inlude, i am brazilian (sorry for any mistake in english, please be nice) and seens like github don't have suport for us in brazil, i cant contact them. I realize the "download zip", but just show the html structure.

https://github.com/deborahsv/Survey-Form-Formul-rio-de-Pesquisa-

**Comment by gurjeetsinghvirdee** [comment, marked as answer]
Hey @deborahsv 
How you can't see the code, it's already there in the repository you provided and it's working too
![image](https://github.com/community/community/assets/73753957/0684c7fe-c6c0-499f-ba44-faefe9d0f57f)

**Reply by deborahsv** [reply]
![Imagem do WhatsApp de 2024-04-24 à(s) 14 46 17_26f8cebd](https://github.com/community/community/assets/158364850/d443548f-4be7-49ac-bf7f-8a6afdc8cedc)

How do you made this, mine look like this, without css :(

**Comment by RFAhmadi** [comment]
Hello. This is your code and the CSS is there.
I don't see any problem unless you mean a private repository and it is another problem.
![photo_2024-04-24_21-09-20](https://github.com/community/community/assets/107707063/19adc457-33f6-4ea3-b5bd-195a8b9b8347)

**Reply by RFAhmadi** [reply]
My English is not good either and I used Google 😅😁

**Reply by deborahsv** [reply]
Yes they are there, but when i open the code, look like this, but the code is right :(

Is good to practice the english lol

![Imagem do WhatsApp de 2024-04-24 à(s) 14 46 17_26f8cebd](https://github.com/community/community/assets/158364850/9a6349b2-3878-4001-b427-a7ecf922439e)

**Reply by gurjeetsinghvirdee** [reply]
are you directly accessing the code from download folder?

**Reply by gurjeetsinghvirdee** [reply]
Open it from your VS code or any IDE you have, you will see the same result as mine

**Reply by deborahsv** [reply]
![Imagem do WhatsApp de 2024-04-24 à(s) 14 58 24_27278344](https://github.com/community/community/assets/158364850/203b9be6-b28a-42ee-8aee-1d8af8b7c1c6)

from here

**Reply by gurjeetsinghvirdee** [reply]
wait wait you don't have to do this, because it's your repo just connect your IDE with your github account

**Reply by deborahsv** [reply]
I knew there was something wrong, how can I see the full code (with css)?

**Reply by gurjeetsinghvirdee** [reply]
which IDE you are using?

**Reply by deborahsv** [reply]
sorry, i didnt see your answer: "Open it from your VS code or any IDE you have, you will see the same result as mine".

**Reply by deborahsv** [reply]
i'll try

**Reply by gurjeetsinghvirdee** [reply]
wait I'll upload a video here may be it help you

**Comment by gurjeetsinghvirdee** [comment]
https://github.com/community/community/assets/73753957/8695a232-2aec-43e6-93e4-5bb61df4d3ab

Go through this @deborahsv

**Reply by deborahsv** [reply]
Thank you very much, I'm starting to learn now, so I have a lot to learn, thank you for your patience. I was thinking that on github you don't need to use vscode. I am so gratefull 🙏🏼🙏🏼🙏🏼

![Imagem do WhatsApp de 2024-04-24 à(s) 15 18 45_f607405c](https://github.com/community/community/assets/158364850/2b1bafbc-f7db-46fb-b8b7-0b5567a9f187)

**Reply by ebndev** [reply]
Thanks @gurjeetsinghvirdee for taking the time to screen record and share!

@deborahsv, if you found that answer helpful, please feel encouraged to "Mark as answer" :)

**Reply by gurjeetsinghvirdee** [reply]
My pleasure
I think you have to watch some tutorial on how to use **github** and how you connect with your vs code even **chat-gpt** will help you too 
You can watch this https://youtu.be/i_23KUAEtUM?si=FaQ867sx5dt_ApOe

**Reply by deborahsv** [reply]
thanks again, i'll watch!!
